machine.go

```diff
@@ -90,21 +137,21 @@ func (cfg *Config) Validate() error {
 	}
 
 	if _, err := os.Stat(cfg.KernelImagePath); err != nil {
-		return fmt.Errorf("Failed to stat kernal image path, %q: %v", cfg.KernelImagePath, err)
+		return fmt.Errorf("failed to stat kernal image path, %q: %v", cfg.KernelImagePath, err)
 	}
 	if _, err := os.Stat(cfg.RootDrive.HostPath); err != nil {
-		return fmt.Errorf("Failed to stat host path, %q: %v", cfg.RootDrive.HostPath, err)
+		return fmt.Errorf("failed to stat host path, %q: %v", cfg.RootDrive.HostPath, err)
```

Comment:
Can we use `errors.Wrap` instead?

Reply:
Major benefit - it keeps call stack info, which sometimes makes life of lot easier.
It's not super important for this specific case, but in general it's better to have this information than not to.

Reply:
I'd prefer to leave it out for now, both because it's an extra dependency that I don't want us to take just yet and because it's out of scope for this PR (I changed one character in that line...).